src/components/forms/PhoneInput.tsx

```diff
@@ -83,7 +83,7 @@ export const PhoneInput = (props: PhoneInputProps) => {
 
```

Comment:
@jmhar3 it looks like this might be because we aren't loading fonts in correctly, which is causing a little visual offset.

I'd love to get this big upgrade in so we can start on a few breaking changes, so let's take care of this visual issue in another PR